go/chat/utils/utils.go

```diff
@@ -860,7 +859,8 @@ func GetDesktopNotificationSnippet(conv *chat1.ConversationLocal, currentUsernam
 	}
 	mvalid := msg.Valid()
 	if !mvalid.IsEphemeral() {
-		return GetMsgSnippet(msg, *conv, currentUsername)
+		snippet, _ := GetMsgSnippet(msg, *conv, currentUsername)
```

Comment:
we should use the decoration at L872 so it's consistent throughout -- why doesn't this snippet need the separation in general?

Reply:
what alignment changes?